android/src/com/mapswithme/maps/routing/RoutingBottomMenuController.java

```diff
@@ -266,15 +266,10 @@ private void showRouteAltitudeChart()
       UiUtils.show(mAltitudeChart);
       String meter = mAltitudeDifference.getResources().getString(R.string.meter);
       String foot = mAltitudeDifference.getResources().getString(R.string.foot);
-      mAltitudeDifference.setText(String.format(Locale.getDefault(), "%d %s",
-                                                limits.maxRouteAltitude - limits.minRouteAltitude,
-                                                limits.isMetricUnits ? meter : foot));
-      Drawable icon = ContextCompat.getDrawable(mContext,
-                                                R.drawable.ic_altitude_difference);
-      int colorAccent = ContextCompat.getColor(mContext,
-          UiUtils.getStyledResourceId(mContext, R.attr.colorAccent));
-      mAltitudeDifference.setCompoundDrawablesRelativeWithIntrinsicBounds(Graphics.tint(icon, colorAccent),
-                                                                  null, null, null);
+	  String unit = limits.isMetricUnits ? meter : foot;
+      mAltitudeDifference.setText(String.format(Locale.getDefault(), "▲ %d %s ▼ %d %s",
+	                                            limits.totalAscent, unit,
```

Comment:
Looks like there are tabs and spaces mixed in the indentation. Please set the editor to use 2 spaces instead of tabs, and check/fix indentation in all your changes.

Reply:
Fixed